definitions/region/connections.yaml

```diff
@@ -0,0 +1,192 @@
+# allow directional connections explicitly
```

Comment:
Region-definitions files have to have a structure like:
```yaml
- <Hierarchy Name>:
    - Region Name
    - ...
```
I suggest "Connections" as hierarchy name.

Reply:
Do the connections belong to this region file then? In open entrance, they are in the mapping folder if I am not mistaken. The regions should be defined already in the definitions, no? I am a bit confused.

Reply:
Yes, but we made some bad design decisions in openENTRANCE and we are implementing a cleaner structure with the new nomenclature package.

The connections file is part of the definitions directory, so it has to follow the structure a region-definition file.
And any region used as origin or destination of a directional connection has to be defined in the region list.

Reply:
I See! I did not know that and thought we should do it like there. Thanks for the info. So I basically list the regions with the hierarchy you mentioned and then do the connections-list below?

Reply:
I'm not sure whether we're talking about the same thing, but update the PR and we'll see.

Reply:
What I meant is that there is a file in the mappings of openentrance that shows the directional connection between nodes. As you said some days ago, I moved that from the mappings folder to the regions folder (here https://github.com/iiasa/idesignres-workflow/pull/3#discussion_r1884021414). What I am wondering now is how to include it in the correct way with the given hierarchy.

Reply:
Please look at the example above.

Reply:
Done, I hope it is correct like this